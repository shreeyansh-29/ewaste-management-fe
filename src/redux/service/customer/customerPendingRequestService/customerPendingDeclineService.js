import api from "../../../../core/utilities/httpProvider";

export const customerPendingDeclineService = async (data) => {
  return api.delete(
    `http://localhost:8083/customer/deleteById?orderId=${data.payload}`
  );
};
