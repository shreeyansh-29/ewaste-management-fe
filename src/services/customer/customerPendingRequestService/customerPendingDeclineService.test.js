/* eslint-disable no-undef */

import {customerPendingDeclineService} from "./customerPendingDeclineService";
import React from "react";
import Enzyme, {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

Enzyme.configure({adapter: new Adapter()});

it("testing customerPendingDeclineService", () => {
  const data = {payload: {id: 1}};
  const Service = customerPendingDeclineService;
  const wrapper = shallow(<Service {...data} />);
  expect.assertions(1);
  expect(toJson(wrapper)).toMatchSnapshot();
});
