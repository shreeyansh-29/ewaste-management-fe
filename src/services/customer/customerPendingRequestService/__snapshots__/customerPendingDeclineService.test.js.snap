// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing customerPendingDeclineService 1`] = `<Component />`;
